common/forms.py

```diff
@@ -136,15 +140,20 @@ class Meta:
             "username",
         ]
 
+    def is_unique_email(self) -> bool:
```

Comment:
Im not a python naming master, but is it not more readable with `email_is_unique`? Is there a reason for putting the `is` first?